contextual_lenses/pfam_family_classification/pfam_transformer_nearest_neighbors_sample_sweep.py

```diff
@@ -0,0 +1,156 @@
+"""Nearest neighbors classification sampling sweep
```

Comment:
there is a considerable amount of shared code with pfam_nearest_neighbors_sample_sweep. Can't you set up an experiment driver that takes a model and runs the big loop. Merging these would be good because then there'd only be one thing to migrate to be caliban-friendly.